### DISCUSSION TITLE
How to know the latest stable version of emqx

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#14277] [Q&A] [answered]
Hi
We are planning to upgrade our older emqx cluster to a new stable version
Is there a way in gitlab release to know which is the lastest stable release?

Thanks in advance!

**Comment by ieQu1** [comment, marked as answer]
Hello,

We don't use gitlab, perhaps you meant github. Releases are listed here: https://github.com/emqx/emqx/releases There's no notion of (un)stable release, the latest release = the latest tag.
